src/SFA.DAS.ApplyService.Types/AppealStatus.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SFA.DAS.ApplyService.Types
```

Comment:
I can't see any references to this in the codebase, is it needed? Or what it previously in one of the deleted files and always there anyway?